### ISSUE TITLE
Management Console Process List should show Business Key as a column

### ISSUE THREAD

**Issue by StephenOTT** [apache/incubator-kie-tools#3655]
### Description

The business key is searchable and it is a required field in the launching of a process instance (as per swagger ui), but when in the mgmt console, you cannot see the ID:

<img width="2654" alt="image" src="https://user-images.githubusercontent.com/1994838/149666774-cfd4aa66-44c6-4eb8-899d-2e7a96365875.png">

### Implementation ideas

Add business key as a column to the process list

**Comment by AjayJagan** [comment]
Hi Stephen, the current design incorporates the business key as a part of the `id`(the first column). It is seen as a blue-colored label adjacent to the process name. To view the id of the process instance,  you can hover over the process name(id column) and it appears as a tool-tip. In any case, I will take it up with the team and have a discussion if we can move it as a separate column.
Thanks.

**Comment by StephenOTT** [comment]
In many cases a business key is a incrementing value. Having it in the column would give the ability to sort?

**Comment by AjayJagan** [comment]
yes if we move it to a column we can sort the table using business keys

**Comment by ricardozanini** [comment]
@pefernan @paulovmr is this still a valid issue?
